portfolio/src/main/java/com/google/sps/data/UserLogin.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+ * Class to keep track of user's email, logout URL, logged in status
+ */
+public class UserLogin {
+  private String email;
+  private String url;
+  private boolean logIn;
+
+  public UserLogin(String email, String logoutURL, boolean logIn) {
```

Comment:
you have a parameter called logoutURL here that's assigned to the url field, but in the other constructor you have the same parameter called loginURL. As far as the class cares, there isn't any difference between a loginURL and a logoutURL, since both are assigned to the url field. So, I'd suggest, just calling these params 'url'

Reply:
I called them logoutURL and loginURL just to clarify why I have two constructors when I look back

Reply:
yeah, unfortunately that's not great style. Since you're putting in the parameter name what that url *has* even though that's how it's used in one instance that you're implementing. It's unnecessarily limiting. 

Does that make sense? If not, let me know and we can chat outside of github on this.